core/ir/x86/decode_table.c

```diff
@@ -1328,154 +1328,155 @@ const instr_info_t * const op_instr[] =
     /* OP_ktestd        */  &vex_W_extensions[105][1],
 
     /* AVX-512 EVEX encoded */
-    /* OP_valignd         */  &evex_W_extensions[154][0],
-    /* OP_valignq         */  &evex_W_extensions[154][1],
-    /* OP_vblendmpd       */  &evex_W_extensions[155][1],
-    /* OP_vblendmps       */  &evex_W_extensions[155][0],
-    /* OP_vbroadcastf32x2 */  &evex_W_extensions[147][0],
-    /* OP_vbroadcastf32x4 */  &evex_W_extensions[148][0],
-    /* OP_vbroadcastf32x8 */  &evex_W_extensions[149][0],
-    /* OP_vbroadcastf64x2 */  &evex_W_extensions[148][1],
-    /* OP_vbroadcastf64x4 */  &evex_W_extensions[149][1],
-    /* OP_vbroadcasti32x2 */  &evex_W_extensions[151][0],
-    /* OP_vbroadcasti32x4 */  &evex_W_extensions[152][0],
-    /* OP_vbroadcasti32x8 */  &evex_W_extensions[153][0],
-    /* OP_vbroadcasti64x2 */  &evex_W_extensions[152][1],
-    /* OP_vbroadcasti64x4 */  &evex_W_extensions[153][1],
-    /* OP_vcompresspd     */  &evex_W_extensions[156][1],
-    /* OP_vcompressps     */  &evex_W_extensions[156][0],
-    /* OP_vcvtpd2qq       */  &evex_W_extensions[46][1],
-    /* OP_vcvtpd2udq      */  &evex_W_extensions[47][1],
-    /* OP_vcvtpd2uqq      */  &evex_W_extensions[48][1],
+    /* OP_valignd         */  &evex_W_extensions[155][0],
```

Comment:
Smaller diff if these are not shifted?  What was the reason to shift?

Reply:
There was no reason other than putting vpermb next to vpermw.